public/index.html

```diff
@@ -19,7 +15,7 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <link href="https://fonts.googleapis.com/css2?family=Fjalla+One&family=Merriweather+Sans:wght@300&display=swap"
+    <link href="https://fonts.googleapis.com/css2?family=Fjalla+One&family=Merriweather+Sans:wght@300;400;700&display=swap"
```

Comment:
question(nb): I'm not quite sure how it works - does importing Fjalla One without any font weights mean we get all the weights? Or what?

Reply:
yes